configuration/pih/petl/jobs/hiv/viral_load/source.sql

```diff
@@ -0,0 +1,126 @@
+SET sql_safe_updates = 0;
+
+DROP TABLE IF EXISTS temp_hiv_constructs;
+DROP TABLE IF EXISTS temp_viral_load;
+DROP TABLE IF EXISTS temp_vl_sample_taken_date;
+DROP TABLE IF EXISTS temp_days_since_vl;
+
+-- hiv constructs table
+CREATE TEMPORARY TABLE temp_hiv_constructs
+(
+patient_id 				INT,
+encounter_id 			INT
+);
+
+INSERT INTO temp_hiv_constructs (patient_id, encounter_id)
```

Comment:
A nitpick, but would this be better as somethign like "temp_hiv_construct_encounters"?

Reply:
Done